packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js

```diff
@@ -183,6 +146,7 @@ export class CheckoutDeliveryOption extends PureComponent {
                 <br />
```

Comment:
Please check whether we can remove this <br>.

Reply:
done